src/components/organisms/AlertMessage.js

```diff
@@ -0,0 +1,76 @@
+import React from "react"
+import styled from "styled-components"
+import { useStaticQuery, graphql } from "gatsby"
+import Alert from "@material-ui/lab/Alert"
+import AlertTitle from "@material-ui/lab/AlertTitle"
+import ContextStore from "@/contextStore"
+
+const AlertMessageContainer = styled.div`
+  && {
+    margin-bottom: 20px;
+  }
+`
+
+const AlertMessage = props => {
+  const config = useStaticQuery(
+    graphql`
+      query {
+        allConfig {
+          edges {
+            node {
+              title_en
+              title_zh
+              message_en
+              message_zh
+              severity
+              variant
+            }
+          }
+        }
+      }
+    `
+  )
+
+  const {
+    allConfig: { edges },
+  } = config
+
+  const {
+    route: {
+      state: { fullPath },
+    },
+  } = React.useContext(ContextStore)
+
+  return (
+    <>
+      {edges.map((edge, index) => {
+        var {
+          node: {
+            title_en,
+            title_zh,
+            message_en,
+            message_zh,
+            severity,
+            variant,
+          },
+        } = edge
+        var title = fullPath.includes("/en") ? title_en : title_zh
+        var message = fullPath.includes("/en") ? message_en : message_zh
```

Comment:
can use withLanguage if you want to select the localized field
`withLanguage(i18n, edge.node, 'title')`

Reply:
@nandiheath in csv, we just need a field called title and message and gatsby will handle it?